Sources/Private/Experimental/Animations/CombinedShapeAnimation.swift

```diff
@@ -0,0 +1,47 @@
+// Created by Cal Stephens on 1/28/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import QuartzCore
+
+extension CAShapeLayer {
+  /// Adds animations for the given `[BezierPath]` keyframes to this `CALayer`
```

Comment:
Does this comment need to be updated? Seems like it takes a `CombinedShapeItem` now

Reply:
this is somewhat right since a `CombinedShapeItem` is a thin wrapper around `[BezierPath]`, but I'll update this to be more clear (you're right that it looks wrong at a glance)